plugins/woocommerce/tests/e2e-pw/tests/customize-store/intro.spec.js

```diff
@@ -115,34 +115,38 @@ test.describe(
 			).toBeVisible();
 		} );
 
-		test( 'it shows the "non default block theme" banner when the theme is a block theme different than TT4', async ( {
-			page,
-		} ) => {
-			await activateTheme( 'twentytwentythree' );
+		test(
+			'it shows the "non default block theme" banner when the theme is a block theme different than TT4',
+			{ tag: '@skip-on-default-pressable' },
+			async ( { page } ) => {
+				await activateTheme( 'twentytwentythree' );
 
-			await page.goto( CUSTOMIZE_STORE_URL );
+				await page.goto( CUSTOMIZE_STORE_URL );
 
-			await expect( page.locator( 'h1' ) ).toHaveText(
-				'Customize your theme'
-			);
-			await expect(
-				page.getByRole( 'button', { name: 'Go to the Editor' } )
-			).toBeVisible();
-		} );
+				await expect( page.locator( 'h1' ) ).toHaveText(
+					'Customize your theme'
+				);
+				await expect(
+					page.getByRole( 'button', { name: 'Go to the Editor' } )
+				).toBeVisible();
+			}
+		);
 
-		test( 'clicking on "Go to the Customizer" with a classic theme should go to the customizer', async ( {
-			page,
-		} ) => {
-			await activateTheme( 'twentytwenty' );
+		test(
+			'clicking on "Go to the Customizer" with a classic theme should go to the customizer',
+			{ tag: '@skip-on-default-pressable' },
```

Comment:
The theme is activated through CLI, which is an option that we don't have right now for Pressable.